libs/application/templates/financial-aid/src/fields/HomeCircumstancesForm/HomeCircumstancesForm.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react'
```

Comment:
Do we want to call these files `....Form` . I know a lot of applications do this, just wondering :)

Reply:
Good point, we name our ennum homecircumstance so maybe form is alright? 😇